tests/integration/test_load_and_serve_agent.py

```diff
@@ -0,0 +1,134 @@
+import asyncio
+import os
+import subprocess
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+
+import requests
+import logging
+
+from concurrent.futures import ThreadPoolExecutor
+
+import pytest
+from litellm.utils import validate_environment
+
+from any_agent import AgentConfig, AgentFramework, AnyAgent, TracingConfig
+from any_agent.config import MCPStdio, ServingConfig
+from any_agent.tracing.trace import AgentTrace, _is_tracing_supported
+
+from queue import Queue
+
+from common.client import A2AClient
+from common.types import TaskSendParams, Message, TextPart
```

Comment:
This import will throw immediately  if someone doesn't have the optional a2a dependency installed,

Reply:
Solved by adding a check in the package-level imports.